src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -72,6 +72,7 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
                     loggingBuilder.AddWebJobsSystem<WebHostSystemLoggerProvider>();
                     loggingBuilder.AddDeferred();
                 })
+                .UseShutdownTimeout(TimeSpan.FromMilliseconds(1)) // Default value is 5 seconds (https://github.com/aspnet/AspNetCore/blob/master/src/Hosting/Hosting/src/Internal/WebHostOptions.cs#L70)
```

Comment:
I don't think we should change this - we want there to be a grace period in general to allow logging and other operations to complete.

Instead, for language worker in flight requests, can't we just handle host shutdown more gracefully?